share/spack/docker/spack_arch/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM pritunl/archlinux
```

Comment:
I appreciate the minimal image, but this does not look too official.
Although there seems not to be an official arch image, there are certainly more popular and automated ones in: https://hub.docker.com/search/?isAutomated=1&isOfficial=0&q=arch we could use?

Reply:
Indeed, there doesn't seem to be an official arch image, and I went with the one I found mentioned in Arch's docker documentation. https://wiki.archlinux.org/index.php/Docker

I remember trying the `base/archlinux` image and having trouble with it, though I can't recall exactly what the issue was.  In any case, I'll give these other images a try and report back.

Reply:
Well, I can't remember what problems I had before, but they seem to be gone, now!  I'll go ahead and push up the change for the arch image's base to `base/archlinux`.